roles/manage-repositories/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Just a general comment on role naming. Given a role when applied is an action itself, typically the naming scheme is to use a noun to represent the thing being managed. For example, instead of `manage-repositories` using `satellite6_repositories`.

Reply:
ok, will update this, I admit me not good with naming stuff :). I like the `noun to represent` guideline.

Reply:
Looking at the role ` manage-repositories` I realized why I gave it the name, it was coz, it had other repos to enable and disable too. But feel better we make it `satellite6-repositories`.